fastestimator/pipeline.py

```diff
@@ -184,11 +186,14 @@ def get_loader(self, mode: str, epoch: int = 0,
             batch_size = self.batch_size
             if isinstance(batch_size, Scheduler):
                 batch_size = batch_size.get_current_value(epoch)
+            if isinstance(data, BatchDataset) and not batch_size is None:
```

Comment:
and batch_size is not None

more readable

Reply:
done